docs/game/cheat_sheet.md

```diff
@@ -60,13 +65,14 @@ Deviation:
 Tactics
 -------
 
-* Speaking in a slow, measured way is beneficial. Just don't slow down to a stop.
+* Speaking in a measured way is beneficial, although be careful not to slow down too much - try and maintain your normal speaking voice and variation in tone.
```

Comment:
Maybe you want to point out that your voice shouldn't come across as distorted on account of you trying to speak slowly.